tests/event.py

```diff
@@ -333,6 +333,19 @@ def test_all_day_outputs_dtstart_value_date(self):
         # no time or tz specifier
         self.assertIn('DTSTART;VALUE=DATE:20151221', str(e).splitlines())
 
+    def test_all_day_tz_independent(self):
+        """
+        see Issue #173
+        """
+        for tz in range(-3, 3):
+            for hour in range(-3, 3):
+                e = Event(begin=Arrow(2019, 05, 29, tzinfo="%+02d:00" % tz).shift(hours=hour))
```

Comment:
```suggestion
                e = Event(begin=Arrow(2019, 5, 29, tzinfo="%+02d:00" % tz).shift(hours=hour))
```
Else it would be considered as octal :)